tests/js/contentAware/contentAware-tests.js

```diff
@@ -80,7 +75,7 @@ fluid.defaults("gpii.express.tests.contentAware.testEnvironment", {
                     express: {
                         port:    "{testEnvironment}.options.port",
                         baseUrl: "{testEnvironment}.options.baseUrl",
-                        views:   viewDir,
+                        views:   "%gpii-express/tests/views",
```

Comment:
We can, it's just a later discussion, hopefully centered around gpii-handlebars.